docs/Setup_Kubernetes_Cluster.md

```diff
@@ -370,6 +370,26 @@ In default Cilium multicast feature is disabled, so we need to walk through the
   cilium-dbg bpf multicast subscriber list all
   ```
 
+- Simplified configuration about Multicast Groups and Subscribers using Cilium CLI
```

Comment:
Can you add what version or later that this feature is available for both cilium and cilium-cli?